frontend/css/brainzplayer.less

```diff
@@ -338,3 +338,58 @@
     }
   }
 }
+
+.queue {
+  position: fixed;
+  padding: 1em;
+  bottom: 60px;
```

Comment:
The animation looks quite smooth now. Instead of adding it in Queues.tsx, I've made the changes in BrainzPlayerUI.tsx since `Queues` is a memorised function, so modifying the state `showQueue` is causing the component to re-render on props change